test/core.js

```diff
@@ -279,6 +283,7 @@ test('xdm', async function (t) {
         {
           components: {
             y: {
+              // @ts-ignore React typings do not support nested component names
```

Comment:
Tracing it back more, it looks like this found an edge case not covered in https://github.com/mdx-js/mdx/blob/39b6f23bef8401b15d0f2f66031647b703952fb3/packages/react/types/index.d.ts#L15-L26

Reply:
Ah. `mdx-js/mdx` does not allow `<a.b />`, this was fixed there and is also supported here`.
Why does that affect this test though?

Reply:
https://github.com/wooorm/xdm/blob/d347c39c923aa71cfd19e6801177f0f22d0e6877/test/core.js#L257

is generic, it will try to match the props defined by

https://github.com/wooorm/xdm/blob/d347c39c923aa71cfd19e6801177f0f22d0e6877/test/core.js#L258

with the props passed here

https://github.com/wooorm/xdm/blob/d347c39c923aa71cfd19e6801177f0f22d0e6877/test/core.js#L259-L265

Reply:
which means the props here, would need to match https://github.com/mdx-js/mdx/blob/39b6f23bef8401b15d0f2f66031647b703952fb3/packages/react/types/index.d.ts#L15-L26
or a `ts-ignore` would need to be applied

Reply:
Ah... OK, that explains it. Well, then this is fine for me!